src/global.d.ts

```diff
@@ -0,0 +1,18 @@
+
+export {};
```

Comment:
what is the use of this export?

Reply:
This is required because TypeScript only allows augmentations to the global scope in module files.

Reply:
@andropixels there must be an issue in `tsconfig`, if you include a `.d.ts` file in your tsconfig you dont have to add an export in a definition file

Reply:
![image](https://user-images.githubusercontent.com/69791440/220877505-d8bf2b75-a798-4b73-98c7-890ac50ad0cd.png)
its there am i missing anything ?

Reply:
look at `https://www.typescriptlang.org/tsconfig#typeRoots`

Reply:
added a fix